### DISCUSSION TITLE
Has the splash_only startup verbosity option been removed?

### DISCUSSION THREAD

**Discussion by PatTheMav** [dosbox-staging/dosbox-staging#2781] [Q&A] [answered]
So far I have always used `splash_only` as my startup verbosity setting but since I switched to 0.81 from CI to test 3dFx support I noticed it’s an unsupported option now and the code doesn’t check for the value anymore but I also couldn’t find a PR or commit with reasoning for it.

**Comment by johnnovak** [comment, marked as answer]
It was dropped a year ago @PatTheMav 
https://github.com/dosbox-staging/dosbox-staging/pull/1842

**Reply by PatTheMav** [reply]
Got it. Might I suggest removing the option from the wiki because it’s still presented there with screenshots and all?

**Reply by johnnovak** [reply]
Good point, we'll do that sometime.

**Reply by PatTheMav** [reply]
I've also looked at the change pondering whether I could just revert it for local builds, but from the look of it I'd rather implement it in a different way (and also not carry the image data around in source), but first I'd need to understand why/how it interferes with pulseaudio.

**Reply by johnnovak** [reply]
Sure, if you can reinstate it cleanly without bugs, with the image being loaded from `resources`, then why not, hey 😄 Out of curiosity, are you really in love with our former splash screen or what's the reason for wanting to bring it back? I'm just curious because I don't think anyone has brought this up before, and we devs don't miss it much either (I definitely don't; the first thing I did was to disable it 😄)

**Reply by PatTheMav** [reply]
It was just a nice touch because I launch DOSBOX-based games from GOG Galaxy with custom configs - I got one config for rectangular-pixel based games, the 320x200 and 640x400 crowd, and another for SVGA-based games and above, then game-specific configs with the `autoexec` block. So in GOG Galaxy I configured "custom executable" with custom command line parameters, specifying the generic and the game-specific configs with `-c "exit"` added.

So when I launch a game, it goes full screen, the DOSBOX splash screen comes up (almost like the PS1 boot screen), then the game starts immediately. Makes DOSBOX almost feel like a "console", which was nice because my PC is part of my home cinema setup, so I play on my telly.

(I also got my emulated games set up a similar way, so `mednafen` is launched directly from GOG Galaxy as well but obviously shows you the entire BIOS boot up process, which is part of the charm).

**Reply by johnnovak** [reply]
Nice one, thanks for explaining. Yeah, I can see the Staging logo coming up as part of the experience in a setup like yours, so it would be nice to have the option to restore it (even if it's off by default). Well, if you could help out, that would be great 😄 That PulseAudio related bullshit seems very surprising, to say the least...

**Reply by PatTheMav** [reply]
I'm primarily on macOS so I'll have to set up a build on one of my Linux VMs, but we implemented PulseAudio in obs-studio as well, so maybe there is someone on the team that can help me there.. 😅
